Need guidance on setting up eslint-plugin-eslint-config

Hi;

I'm trying to enable the "eslint-config/no-unknown-rules" rule, but not having much luck.

* Read the [Readme](https://github.com/G-Rath/eslint-plugin-eslint-config/blob/master/README.md) top-to-bottom several times, and even though I've got a completely bogus rule ("xyzzy"), running doesn't say anything about that unknown rule:
```bash
eslint .eslintrc.json -c .eslintrc.json --ext .json --debug
```
* Created an .eslintignore file:
```
# eslint ignores hidden files by default
!.*.json
**/node_modules
```
* Created a bare-bones .eslintrc.json file:
```json
{
  "plugins": [ "eslint-config" ],
  "overrides": [
    {
      "files": [ ".eslintrc.js", ".eslintrc.json", "react.js", "@typescript-eslint.js" ],
      "rules": {
        "eslint-config/no-unknown-rules": "error",
        "foo-bar/baz": "warn"
      }
    }
  ],
  "rules": {
    "xyzzy": "error"
  }
}
```
* When run, the output is:
```

/home/ubuntu/work/ns/management-ui/.eslintrc.json
  2:12  error  Parsing error: Unexpected token :

✖ 1 problem (1 error, 0 warnings)
```
* Version information:
```
$ yarn list --pattern eslint 

yarn list v1.22.10
├─ @types/eslint-visitor-keys@1.0.0
├─ @typescript-eslint/eslint-plugin@2.34.0
├─ @typescript-eslint/experimental-utils@2.34.0
├─ @typescript-eslint/parser@2.34.0
├─ @typescript-eslint/scope-manager@4.31.2
├─ @typescript-eslint/types@4.31.2
├─ @typescript-eslint/typescript-estree@2.34.0
├─ @typescript-eslint/visitor-keys@4.31.2
│  └─ eslint-visitor-keys@2.1.0
├─ babel-eslint@10.1.0
├─ eslint-config-airbnb-base@13.2.0
├─ eslint-config-airbnb@17.1.1
├─ eslint-config-prettier@4.3.0
├─ eslint-config-react-app@5.2.1
├─ eslint-import-resolver-node@0.3.6
├─ eslint-loader@3.0.3
├─ eslint-module-utils@2.6.2
├─ eslint-plugin-eslint-config@2.0.0
│  ├─ @typescript-eslint/experimental-utils@4.31.2
│  ├─ @typescript-eslint/typescript-estree@4.31.2
│  ├─ eslint-utils@3.0.0
│  └─ eslint-visitor-keys@2.1.0
├─ eslint-plugin-flowtype@4.6.0
├─ eslint-plugin-import@2.24.2
├─ eslint-plugin-json-format@2.0.1
├─ eslint-plugin-jsx-a11y@6.4.1
├─ eslint-plugin-prettier@3.4.1
├─ eslint-plugin-react-hooks@1.7.0
├─ eslint-plugin-react@7.26.0
├─ eslint-scope@5.1.1
├─ eslint-utils@2.1.0
├─ eslint-visitor-keys@1.3.0
├─ eslint@6.8.0
│  └─ eslint-utils@1.4.3
├─ react-scripts@3.4.4
│  ├─ eslint-plugin-import@2.20.1
│  ├─ eslint-plugin-jsx-a11y@6.2.3
│  └─ eslint-plugin-react@7.19.0
└─ webpack@4.42.0
   └─ eslint-scope@4.0.3
```
What am I doing wrong??

Thanks

--- a/README.md
+++ b/README.md
@@ -48,6 +48,9 @@ Then configure the rules you want to use _for your config(s)_ using `overrides`:
 ```
 
 The rules assume that the files they're linting are configs meant for ESLint.
+Also note that while ESLint can be configured using JSON and YAML files, it
+doesn't support linting those file types meaning this plugin will not work if
+you're not using a `.js` config file.
 
 ## Shareable configurations
 

